DelfinPerformance/README.md

```diff
@@ -0,0 +1,67 @@
+## Delfin Performance Monitoring with Preometheus
+
+This is demo of SODA Foundations Delfin performance monitoring feature using default preometheus exporter.
+
+
+Delfin is a heterogeneous infrastructure management platform. It manages and monitors multiple storage backends from a single place. Prometheus is default integrated with delfin for monitoring performances of the storages.
+
```

Comment:
Please provide a section with complete information about this example:
What is this example 
When you run, what you can see/verify.
Why this example?
etc...

Reply:
done